vars/archiveAssembleUpload.groovy

```diff
@@ -1,4 +1,5 @@
 /*
+* Copyright OpenSearch Contributors
```

Comment:
Can you fix the indentation for all groovy files?
Thanks!

Reply:
Sure Sayali.